imsim/readout.py

```diff
@@ -270,16 +269,31 @@ def get_primary_hdu(eimage, lsst_num, camera_name=None,
         telcode = 'CC'
     else:
         phdu.header['FILTER'] = LSSTCam_filter_map.get(band, None)
-        phdu.header['INSTRUME'] = 'LSSTCam'
+        if camera_name == 'LsstCamSim':
+            phdu.header['INSTRUME'] = 'LSSTCamSim'
+        else:
+            phdu.header['INSTRUME'] = 'LSSTCam'
         phdu.header['RAFTBAY'] = raft
         phdu.header['CCDSLOT'] = sensor
         phdu.header['RA'] = ratel
         phdu.header['DEC'] = dectel
         phdu.header['ROTCOORD'] = 'sky'
+        phdu.header['ROTPA'] = rotang
         telcode = 'MC'
     dayobs = eimage.header['DAYOBS']
     seqnum = eimage.header['SEQNUM']
     contrllr = eimage.header['CONTRLLR']
+    phdu.header['TELESCOP'] = SIMONYI_TELESCOPE
```

Comment:
Maybe factor out some of the other common keywords too?  E.g., 'RAFTBAY', 'CCDSLOT', 'RA', 'DEC', 'ROTCOORD', 'ROTPA', ...?

Reply:
In order to do that more cleanly, I'd like to get rid of support for `LsstCamImSim`, otherwise I suspect the number of `if` statements will actually increase.  In fact, in addition to getting rid of `LsstCamImSim` support, I'd like to support just `*Sim` cameras explicitly, e.g., `LsstCamSim` and `LsstComCamSim`, so also removing `LsstCam` as an option.  I'll post an issue with this proposal.